charts/apisix/Chart.lock

```diff
@@ -0,0 +1,12 @@
+dependencies:
+- name: etcd
+  repository: https://charts.bitnami.com/bitnami
+  version: 5.2.1
+- name: apisix-dashboard
+  repository: https://charts.apiseven.com
+  version: 0.1.2
+- name: apisix-ingress-controller
+  repository: https://charts.apiseven.com
+  version: 0.4.0
+digest: sha256:111cf0f06c6039a02340f3ddafba83882336774ec5539dfceef8f6d59d5d6d34
+generated: "2021-05-11T15:52:54.405811637+08:00"
```

Comment:
This `.lock` file is an intermediate file,  don’t need to submit it.

Suggest noting users  in the document to update dependencies.

Reply:
I think we should keep the Chart.lock file. This is a way to ensure that all users can get the same dependencies when using our Helm Chart. 
I refer to the helm repository of prometheus, which is currently widely used.  https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/Chart.lock

Reply:
This was introduced at https://github.com/apache/apisix-helm-chart/pull/74/commits/3cba3534be2792e46b628ee5e09c876d44bbce37, if you think this is unreasonable, then I can easily discard it.

Reply:
cc @tokers

Reply:
I think we should keep it.

Reply:
ok